Plugins/JiraCommitTipPlugin/PasswordSetting.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Windows.Forms;
+using GitUIPluginInterfaces;
+
+namespace JiraCommitPlugin
+{
+    public class PasswordSetting : ISetting
```

Comment:
What's the point of duplicating _Plugins/GitUIPluginInterfaces/Settings/PasswordSetting.cs_?
It seems that this file can be safely removed and it's usages switched to the common implementation provided by the Api.